crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -200,14 +194,24 @@ impl ParsedCxxQtData {
     }
 
     fn try_parse_inherit_verbatim(&mut self, tokens: TokenStream) -> Result<Option<Item>> {
```

Comment:
Needs to be renamed to `try_parse_verbatim`, as it's no longer limited to `inherit`.